src/styles/globals.scss

```
@@ -113,5 +113,5 @@ button {
 
 input:-webkit-autofill,
 input:-webkit-autofill:focus {
-  transition: background-color 600000s 0s, color 600000s 0s;
+  transition: background-color 600000s 0s, color 0s 0s;
```

```suggestion
  transition: background-color 600000s 0s;
```

You can delete the color portion completely to let it transition normally and leave the background color css